assets/src/edit-story/karma/media.cuj.karma.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Maybe we should put them all into `karma/cuj/` dir?

Reply:
I don't think so. I think all integration tests will test one (or more) CUJ's and should still be co-located with the code in question. So I think all integration/karma tests should have this CUJ annotation.

Reply:
Yeah I've only added these two `*.cuj.karma.js` files for demo/testing purposes. Moving them to where the code sits makes sense to me, though that can happen in subsequent PRs.